compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -141,10 +141,11 @@ gather_builtincode_map(const ::tflchef::ModelRecipe &model_recipe)
 
   for (const auto &operation : model_recipe.operation())
   {
-    auto op_chef = op_chef_registry().lookup(operation.type()).create(&operation);
-    if (op_chef->code() == tflite::BuiltinOperator_CUSTOM)
+    if ((operation.has_extype() && operation.extype() == "Custom") ||
+        (operation.has_type() && operation.type() == "Custom"))
       continue;
```

Comment:
First of all, if there is a `extype` or the type is `Custom`, the code is written to determine that it is a custom op.
--> Logic to refer to (https://github.com/Samsung/ONE/pull/11750#discussion_r1368165801)
If the `extype` in recipe files is removed in step `3)`, everything related to the `extype` will be deleted again in the step `4)`